src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,15 @@
 import React from 'react';
+import { PreparedPosts } from '../../types/PreparedPosts';
+import { PostInfo } from '../PostInfo';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+interface Props {
+  posts: PreparedPosts[];
```

Comment:
```suggestion
  posts: PreparedPost[];
```

Reply:
But I have interface named PreparedPosts. Why  PreparedPost is better. I received many posts

Reply:
Bc if you want to take 1 post, what you will do?
post: PreparedPosts ?
Why one post, but the type is named in the plural?
One post - PreparedPost
Array of post - PreparedPost[]